StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSavedPaymentMethodsCollectionViewController.swift

```diff
@@ -28,6 +28,7 @@ protocol CustomerSavedPaymentMethodsCollectionViewControllerDelegate: AnyObject
         viewController: CustomerSavedPaymentMethodsCollectionViewController,
         paymentMethodSelection: CustomerSavedPaymentMethodsCollectionViewController.Selection,
         updateParams: STPPaymentMethodUpdateParams) async throws -> STPPaymentMethod
+    func didDismiss(viewController: CustomerSavedPaymentMethodsCollectionViewController)
```

Comment:
The only place that didDismiss is implemented (before this change) is in the EmbeddedPaymentElement+Internal and it just closes the sheet btw.

Reply:
So are you saying that instead of didDismiss, I rename that to shouldCloseSheet and in each didDismiss func, I only do presentingViewController?.dismiss(animated: true) or something instead of things like complete() or didCancel()?

Reply:
Partially, not suggesting a change to the implementation, just changing the name `didDismiss` throughout the PR, I don't think `didDismiss` is accurate, b/c technically the sheet dismisses in other scenarios, and in EmbeddedPaymentElement+Internal we need someone else to dismiss us. What we're really doing is dismissing and asking PaymentSheet/embedded to dismiss.